src/challenges/challengeOne/1.Accordion/Accordion.test.tsx

```diff
@@ -1,8 +1,21 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Accordion } from "./Accordion";
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { Accordion, AccordionProps } from "./Accordion";
 
-// describe("Accordion", () => {
-//   it("renders...", () => {});
-// });
+describe("Accordion", () => {
+  //   const onClick = jest.fn();
```

Comment:
Starting off, you never want to leave commented-out code in your files, it hurts maintainability. Remember that everything is in source control and the history of files. So deleting it even when you didn't write it is a good habit to get into.